util/asm_dev/armv8/p256/Makefile

```diff
@@ -0,0 +1,42 @@
+#------------------------------------------------------------------------------------
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#------------------------------------------------------------------------------------
+
+EXE = p256
+
+SRCDIR = ./src
```

Comment:
Nit: adding some references and comments in place would be helpful for later maintainer

Reply:
Discussed and Ack.